### PR TITLE
Fix navbar-burger color when color modifier is used

### PR DESCRIPTION
**Pull request by corneliusludmann**
When using a color modifier for a navbar (e.g., is-dark) the burger icon does not change the color. This commit adds the color modification for the navbar burger icon.

This is a **bugfix**.
Issue #1981.

### Proposed solution

This solution adds color-invert for the .navbar-burger for the color modifiers the same way it is done for .navbar-link.

### Tradeoffs

No tradeoffs.

### Testing Done

I checkout out master and changed the lines. The color of the burger icon is as desired. You can see a minimal example I used there: https://jsfiddle.net/pm26zwer/6/

See also issue #1981.

### CODE REVIEW

File: sass/components/navbar.sass [hunk 1]
```diff
@@ -74,6 +74,8 @@ $navbar-bottom-box-shadow-size: 0 -2px 0 0 !default
         .navbar-link
           &::after
             border-color: $color-invert
+        & > .navbar-burger
```
Comment (jgthms) [inline sass/components/navbar.sass]: You don't really need the nesting here. Just use `.navbar-burger`.

Comment (corneliusludmann) [reply sass/components/navbar.sass]: OK, I changed it. Is it that what you meant?

Comment (jgthms) [reply sass/components/navbar.sass]: Not exactly. Put `.navbar-burger` as a sibling of `.navbar-brand`, not as a child of `.navbar-brand`.

Comment (corneliusludmann) [reply sass/components/navbar.sass]: Ah, I see: `.navbar-burger` does not necessarily need to be a child of `.navbar-brand`. Makes sense. So, here my next try. Is that correct? Thank's for your patience.

**Comment by corneliusludmann** [comment]
Just want to ask whether it is ready to merge after the latest correction to include it in the next release!? Is there anything I can do to help? Thanks for considering …

**Pull request merged**